manual_scripts/logs/download-alb-logs.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Script to download ALB logs from S3
+
+DLUHC_ENV=staging
+APP=cpm
+ACCOUNT_ID=486283582667
+
+FOLDER="${APP}-${DLUHC_ENV}-alb-logs"
+
+mkdir -p "${APP}-alb-logs"
+
+# Change the date in the S3 URI
+aws s3 cp --recursive "s3://${DLUHC_ENV}-${APP}-alb-access-logs/${DLUHC_ENV}-${APP}-alb/AWSLogs/${ACCOUNT_ID}/elasticloadbalancing/eu-west-1/2023/06/" "${FOLDER}/"
```

Comment:
I feel like it'd be neater to make the date a variable like the account id etc. 
And you'd probably have the day in there more often than not?